### DISCUSSION TITLE
Notes for September 26th meeting

### DISCUSSION THREAD

**Discussion by FranckCo** [linked-statistics/COOS#96] [General]
### Update on issues

* Issue #88: pull request made with additional text, reviewed and merged (see https://github.com/linked-statistics/COOS/commit/ca120aadfacfc41e3855b822d42ca7964398d9a0 for details)

* Issue #89: still some discussion on the text, see [pull request](https://github.com/linked-statistics/COOS/pull/91)

* Issue #84 assigned to Florian: write a "COOS use case" document

* Issue #81 assigned to Florian and tagged for version 2. Issue #97 created for one of the questions, subject should be clarified for the other issue to create ("link between statistical data set and product based on GSIM revision")

* Issue #78

A first version of the `ReferenceDocument` class is provided in commit https://github.com/linked-statistics/COOS/commit/c9ded05695d5efc30b989ef27581cde25a72c369. Note the following:
. The singular is used instead of the proposed `ReferenceDocuments`. Each document is a specific resource, so it is preferable to link the documents individually.
. `coos:ReferenceDocument` is defined as a subclass of `foaf:Document`
. the Spanish translations are missing

A first version of the `informs` property is provided in commit https://github.com/linked-statistics/COOS/commit/1a49994d393d4116b76bdf312dbcaaeb2b509439. Note the following:
. The domain of the property is `coos:ReferenceDocument` and its range is `coos:Activity`
. Multiple occurrences of the property can be used to link multiple instances of `coos:ReferenceDocument` (see above)
. The Spanish translations are missing

[Pull request](https://github.com/linked-statistics/COOS/pull/94) created and submitted to Flavio and Zoltán

Comment by Flavio: How about "To provide supporting information for performing an activity"? -> OK, commit added to the PR

Propose to add text at the end of 3.3: an example would be useful

Expecting Zoltan's review

* Issue #76: do we submit for NTTS? Who?

* Issue #72: assigned to Christine and tagged for version 2

* Issue #69: assigned to Franck (to set up dedicated meeting) and tagged for version 2

* Issue #65: assigned to Edgardo

* Issue #63: assigned to InKyung

* Issue #60: Flavio contributed, to discuss

* Issue #59: assigned to Franck, waiting for other issues

* Issue #19: to discuss

* Issue #4 closed

### Ontology expert feedback

* Documenting the source standard for the different items ([ref](https://github.com/linked-statistics/COOS/wiki/Review-of-COOS-model---Thomas-Francart---2021-09-16#documentation)): still to do.

* OWL equivalence between class and ConceptScheme members for product content and product presentation ([ref.](https://github.com/linked-statistics/COOS/wiki/Review-of-COOS-model---Thomas-Francart---2021-09-16#productcontent--productpresentation)): pull request created, reviewed and merged, see commit https://github.com/linked-statistics/COOS/pull/93/commits/1fad04f59abde13cd6448dbed12c73f33fe6c31f for details.

* Application profile ([ref.](https://github.com/linked-statistics/COOS/wiki/Review-of-COOS-model---Thomas-Francart---2021-09-16#application-profile)): issue to create at tag for version 2.

* Activity/Activity category ([ref.](https://github.com/linked-statistics/COOS/wiki/Review-of-COOS-model---Thomas-Francart---2021-09-16#activity--statisticalactivity)): work in progress by Florian, greater impact than initially thought

### Public review

We might have to postpone by a week (see previous point)

**Comment by InKyungChoi** [comment]
Regarding public review: I went through email list used for GSIM revision and GSBPM revision, not sure if the profile is right. Any suggestions (on top of those already made: contact@sdmx.org, twg@sdmx.org and swg@sdmx.org, gonzalezmorales@un.org) would be appreciated!

**Reply by FlavioRizzolo** [reply]
On the DDI community, we might want to include Arofan Gregory (ilg21@yahoo.com), chair of the DDI CDI working group, and who is also with CODATA leading the development of the cross-domain interoperability framework (CDIF). Other people unrelated to the official stats world but that might have something interesting to say are Wendy Thomas (wlt@umn.edu), chair of the DDI Technical Committee, and Jon Johnson (spuddybike@gmail.com), who is a member of the DDI Technical Committee, metadata practitioner and organizer of EDDI, the European DDI Conference. Wendy and Jon could provide a different take on COOS, I think.

**Reply by InKyungChoi** [reply]
List so far

**Expert reviewer (first round) + recommendations**
•	Jay Greenfield, [nightcleaner@gmail.com](mailto:nightcleaner@gmail.com) 
•	Matjaz, [m.jug@cbs.nl](mailto:m.jug@cbs.nl)
•	Robert McLellan, [robert.mclellan@canada.ca](mailto:robert.mclellan@canada.ca) 
•	? Monica Scannapieco, [scannapi@istat.it](mailto:scannapi@istat.it)
•	Rosemary McGrath,  rosemary.mcgrath@stats.govt.nz
•	[contact@sdmx.org](mailto:contact@sdmx.org), [twg@sdmx.org](mailto:twg@sdmx.org) and [swg@sdmx.org](mailto:swg@sdmx.org)
•	[gonzalezmorales@un.org](mailto:gonzalezmorales@un.org)
•	Arofan Gregory, [ilg21@yahoo.com](mailto:ilg21@yahoo.com)
•	Wendy Thomas, [wlt@umn.edu](mailto:wlt@umn.edu)
•	Jon Johnson, spuddybike@gmail.com

**SSG members** 
•	Zoltán Vereczkei - Chair	Hungary
•	Anna Dlugosz	Poland
•	Cory Chobanik	Canada
•	David Barraclough	OECD
•	Essi Kaukonen	Finland
•	Flavio Rizzolo	Canada / DDI
•	Florian Vucko	France
•	Dan Gillman	United States
•	Edgardo Greising	ILO
•	Juan Muñoz	Mexico
•	Kevin McCormack	Ireland
•	Franck Cotton	France
•	Martina Hahn 	Eurostat
•	Mauro Bruno	Italy
•	Omurbek Ibraev	Kyrgyzstan
•	Waleed Mohamed	Egypt
•	Matjaz Jug	Netherlands
•	Carlo Vaccari	Italy
•	José de Jesús Luján Salazar	Mexico
•	Juan Eduardo Rioja Romo	Mexico
•	Manuel Cuellar Rio	Mexico
•	Andrea Petres	Hungary
•	Janusz Dygaszewicz	Poland
•	Federico Segui 	Uruguay 
•	Daniel Blanc 	Uruguay 
•	Helda Mitre	Albania

**Comment by InKyungChoi** [comment]
Meeting notes (September 26th)

Participants: Franck Cotton, Jenny Linnerud, Flavio Rizzolo, Michele Riccio, Zoltan Vereczkei, Juan Munoz, Edgardo Greising, Florian Vucko, Anna Długosz, Paolo Francescangeli, Dan Gillman and InKyung Choi

[Issue #81 Comments on datasets in relation to DCAT](https://github.com/linked-statistics/COOS/issues/81)
- Regarding issue I: although underlying data values might be the same, the contents in different formats (e.g., wide format vs. long format) are not necessarily same semantically. It is not just that we move content from data to metadata (or vice versa). The difference is also semantic not just structure between different formats. For example, qualifying attribute in the wide format applies to the whole entry (record), while in the long format, we can say at more granular level. Action: to document somewhere in issue about these differences and subtleties. 
- Regarding issue II: In GSIM, Product "consumes" Data Set. In COOS, we say coos:dataSet is a sub-class of coos:product. We need to make sure if there is no inconsistency. Action: to open an issue with clarification

[Issue #78 Use Case: link information resources (methodologies) and business process](https://github.com/linked-statistics/COOS/issues/78): Action: [@Vereczkei Zoltán](mailto:Zoltan.Vereczkei@ksh.hu) to provide examples of reference document 

[Issue #76 Promotion of COOS](https://github.com/linked-statistics/COOS/issues/76): ISI session is approved. Idea to organise a session during NTTS - to be further discussed in the Supporting Standards Group meeting

[Issue #60 Inclusion of Statistical Support Program](https://github.com/linked-statistics/COOS/issues/60)
- Flavio made a proposal where we have "production phase" and "overaching activity" instead of GSBPM phases. 
- It would be better to separate "change phase" (GSBPM 1-3) and other GSBPM overarching processes in “overarching activity”. Action: [@Inkyung Choi](mailto:choii@un.org) to make a comment on github issue

[Issue #65 coos:StatisticalOrganization and regional statistical offices](https://github.com/linked-statistics/COOS/issues/65)
- It is important to widen the scope to include those in national statistical system, not just national statistical organisations
- Edgardo made a proposal Action: to continue discussion on github issue

[Issue #63 Classify GAMSO activities](https://github.com/linked-statistics/COOS/issues/63): Action: [@Gillman, Daniel - BLS](mailto:Gillman.Daniel@bls.gov) to make a comment on the issue about "Manage Statistical Methodology" being statistical activity

[Ontology expert review](https://github.com/linked-statistics/COOS/wiki/Review-of-COOS-model---Thomas-Francart---2021-09-16)
- Action: [@Cotton Franck](mailto:franck.cotton@insee.fr) to open an issue about application profile and tag for version 2
- The impact of differentiating activity and activity category is bigger than expected. Would need to postpone the public review by one week.
